src/test/scala/it/pagopa/interop/purposeprocess/SpecHelper.scala

```diff
@@ -75,6 +77,21 @@ trait SpecHelper extends SprayJsonSupport with DefaultJsonProtocol with MockFact
       .once()
       .returns(Future.successful(result))
 
+  def mockRiskAnalysisServiceNoPA = {
+    (() => mockRiskAnalysisService.riskAnalysisForms()).expects().once().returns(SpecData.riskAnalysisServiceNoPA)
+    (() => mockRiskAnalysisServiceSupplier.get()).expects().once().returns(mockRiskAnalysisService)
+  }
+
+  def mockRiskAnalysisServiceCompleteVersion1 = {
+    (() => mockRiskAnalysisService.riskAnalysisForms()).expects().once().returns(SpecData.riskAnalysisServiceComplete)
+    (() => mockRiskAnalysisServiceSupplier.get()).expects().returning(mockRiskAnalysisService).once()
+  }
+
+  def mockRiskAnalysisServiceEmptyVersion = {
+    (() => mockRiskAnalysisService.riskAnalysisForms()).expects().once().returns(SpecData.riskAnalysisServiceNoVersion)
+    (() => mockRiskAnalysisServiceSupplier.get()).expects().returning(mockRiskAnalysisService).once()
+  }
+
```

Comment:
do not mock these, instead let's verify in the test that it is returning the real values that we expect
```suggestion
```